README.md

```diff
@@ -62,8 +64,8 @@ Documentation should be modified within `main.tf` and generated using [terraform
 Generate them like so:
 
 ```bash
-go get github.com/segmentio/terraform-docs
-terraform-docs md ./ | cat -s | tail -r | tail -n +2 | tail -r > README.md
+go get github.com/getcloudnative/terraform-docs
```

Comment:
Why would we switch to this random forked repo for `terraform-docs`?

Reply:
At least this fork doesn't have _"This project currently has no active maintainers"_ label.
I wanted features included in this PR: https://github.com/segmentio/terraform-docs/pull/53
However, if you don't like it, I don't force this change.

Reply:
> "This project currently has no active maintainers" 

Ahh I see!

Reply:
Hi, I am the author of the afore-mentioned PR. Unfortunately, folks at segmentio have not been  responsive to my suggestion of having me join as a maintainer 🤞

Reply:
Also, the source of this change should happen within `main.tf` or it'll be blown away down the road.

Reply:
Ok, I will restore it. However there is a problem with GNU `tail`:
```bash
$ tail -r
tail: invalid option -- 'r'
Try 'tail --help' for more information.

$ tail --version
tail (GNU coreutils) 8.28
...

$ uname -a
Linux meet18core 4.15.0-32-generic #35-Ubuntu SMP Fri Aug 10 17:58:07 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```
How about `| cat -s | tac | tail -n +2 | tac`?
Test:
```diff
$ terraform-docs md ./ | cat -s | tac | tail -n +2 | tac | diff -u README.md -
--- README.md   2018-08-24 01:31:51.699898157 -0500
+++ -   2018-08-24 01:32:59.508426197 -0500
@@ -92,7 +92,6 @@
 
 MIT Licensed. See [LICENSE](https://github.com/terraform-aws-modules/terraform-aws-eks/tree/master/LICENSE) for full details.
 
-
 ## Inputs
 
 | Name | Description | Type | Default | Required |
@@ -135,4 +134,3 @@
 | worker_security_group_id | Security group ID attached to the EKS workers. |
 | workers_asg_arns | IDs of the autoscaling groups containing workers. |
 | workers_asg_names | Names of the autoscaling groups containing workers. |
-
```
Is there `tac` in FreeBSD/Mac?

Reply:
> Why would we switch to this random forked repo for `terraform-docs`?

I am happy to inform you that `segmentio/terraform-docs` is again under active maintenance. The features in `getcloudnative/terraform-docs`, including the one that generates default values for aggregate types (which has been emphasized in the discussion further down below at https://github.com/terraform-aws-modules/terraform-aws-eks/pull/95#discussion_r212485763), have been merged into `master`. The option you want to look out for is `--with-aggregate-type-defaults`.

Reply:
Great, thanks for the update @metmajer !

I see that 0.4 was just released but this isn't available via brew. Is that correct?

Reply:
That's correct, I'm still working on this. The PR to brew is planned to go out this week.

Reply:
It took a bit longer than expected, but we do have a `HomebrewFormula/terraform-docs.rb` as part of the project, which will be updated with each release and contributed to the upstream github.com/Homebrew/homebrew-core` repository. Since right now, we have the latest v0.4.5 available as [binary releases](https://github.com/segmentio/terraform-docs/releases/tag/v0.4.5) and via `brew`.